main/tests/test_astar.cpp

```diff
@@ -102,9 +102,9 @@ bool test_add_remove() {
 	a.connect_points(1, 3, true);
 	a.connect_points(1, 4, false);
 
-	ok = ok && (a.are_points_connected(2, 1) == true);
-	ok = ok && (a.are_points_connected(4, 1) == true);
-	ok = ok && (a.are_points_connected(2, 1, false) == true);
+	ok = ok && (a.are_points_connected(2, 1));
+	ok = ok && (a.are_points_connected(4, 1));
+	ok = ok && (a.are_points_connected(2, 1, false));
 	ok = ok && (a.are_points_connected(4, 1, false) == false);
```

Comment:
I gather your script only handles `== true` but not `== false`?

Reply:
`== false` can't be easily replaced in-place, but `== true` can always be deleted in my experience.